.shopify-build/setup-consumer-test.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+shopt -s extglob
+set -e
+
+ls -l
+mkdir ../tmp/polaris-react
+mv !(node_modules) ../tmp/polaris-react
+mv ../tmp/polaris-react polaris-react
+git clone ssh://git@github.com/Shopify/$1 --depth 1
```

Comment:
Curious if this has to be `ssh://git@github.com/...` or if `https://github.com/...` would also work?

Reply:
It doesn't, CI can't authenticate with HTTP the only thing it has is the SSH key